package.json

```diff
@@ -22,12 +22,13 @@
   },
   "main": "node/index.js",
   "dependencies": {
+    "babel-core": "^5",
     "babel-runtime": "^5",
     "lodash": "^3.3.1",
+    "through": "^2.3.6",
     "tiny-sprintf": "^0.3.0"
   },
   "devDependencies": {
-    "babel-core": "^5",
```

Comment:
Ah I see. I had a hunch that this is the problem. Browserify picks up the `browserify` field in `package.json`.

I think there are two reasonable solutions:

- Instead of using the `browserify` field, specify the transforms in the CLI (i.e. in the commands in the `scripts` field).
- (this may be the better solution) Run browserify over the `dist/browser.js` instead of `src/browser.js`. 

More explanation regarding the second option: 
Currently there are two entry points for the transform: `scripts/build-node.js` and `browserify`. The former transforms each file and copies it to `dist/`. The latter transforms each file *and* bundles them together. But in both cases the [transformations are the same](https://github.com/fkling/JSNetworkX/blob/master/transforms/es7.js#L23-L48). So, instead of specifying a transform for browserify, we could just run it on the already transformed code (which of course requires to run the transform beforehand, separately).

We might still need both entry points, so that `npm run watch:browser` works, but it's probably still an improvement.